config/locales/js-en.yml

```diff
@@ -399,6 +399,8 @@ en:
     label_expand_project_menu: "Expand project menu"
     label_export: "Export"
     label_export_preparing: "The export is being prepared and will be downloaded shortly."
+    label_file_link_no_permission: "No permission to see this file."
```

Comment:
If these are user shown error messages, I'd prefer a correct sentence like "You have no permission to see this file". Same for the other string.

Reply:
This was the message defined by this figma file: https://www.figma.com/file/gtLQfPe09X7XugAH8L7dTy/Nextcloud-Integration?node-id=1062%3A33627

I will contact Marc to discuss those texts.